src/Files.App/ViewModels/Properties/FileProperties.cs

```diff
@@ -114,7 +115,17 @@ public override async void GetSpecialProperties()
 				ViewModel.ItemCreatedTimestamp = Item.ItemDateCreated;
 				ViewModel.ItemAccessedTimestamp = Item.ItemDateAccessed;
 				ViewModel.LoadLinkIcon = Item.LoadWebShortcutGlyph;
-                ViewModel.IsRunAsAdmin = ((ShortcutItem)Item).RunAsAdmin;
+
+				var shortcutItem = (ShortcutItem)Item;
+
+                var isApplication = !string.IsNullOrWhiteSpace(shortcutItem.TargetPath) &&
```

Comment:
I think that putting this in GetBaseProperties() is enough. No need to do this twice.

Reply:
Oh true